addons/sale_timesheet/models/project.py

```diff
@@ -555,12 +553,6 @@ def _compute_analytic_account_active(self):
         for task in self:
             task.analytic_account_active = task.analytic_account_active or task.so_analytic_account_id.active
 
-    @api.depends('allow_billable')
-    def _compute_sale_order_id(self):
-        billable_tasks = self.filtered('allow_billable')
-        super(ProjectTask, billable_tasks)._compute_sale_order_id()
-        (self - billable_tasks).sale_order_id = False
-
     @api.depends('commercial_partner_id', 'sale_line_id.order_partner_id', 'parent_id.sale_line_id', 'project_id.sale_line_id', 'allow_billable')
     def _compute_sale_line(self):
```

Comment:
`About moving this method to sale_project`
May lead to many changes in tests of sale_project because many tasks in tests are private non billable .
should I change tests of sale_project and move this to sale_project?
(mostly this [file](https://github.com/odoo/odoo/blob/master/addons/sale_project/tests/test_child_tasks.py) SO is in draft state and service does not create any project)